README.md

```diff
@@ -1,2 +1,22 @@
-hello world
+# The Team 7 Project (Name TBA)
 
+A website that connects the homeless to shelter.
+
+## Running Locally
+
+Redirect to ./App and run:
+
+```bash
+mvn package exec:java
+```
+The application should then run on port 8080.
+
+
+## Team 7
+[Perry Cate](https://github.com/perrycate)
```

Comment:
Bah put me at the bottom and add some parenthesis like "advisor" or something. Y'all are the ones doing the real work ;)